### DISCUSSION TITLE
Latest update has caused issue in our app builds

### DISCUSSION THREAD

**Discussion by Sham-Hulla** [vuejs/test-utils#2130] [General]
Hi all,
Not sure what the right forum for this is but I thought I'd flag this and it might help others who could be affected by the same issue.

The latest change to this project(https://github.com/vuejs/test-utils/commit/136e67b2466f995ff25688f06f751b64818f6cf3) has upgraded a dependency(prettier) to a version that uses a more recent version of node and because our codebase is still on a pretty old and unsupported version(14.21.x) it's stopped building a bunch of our apps:
```Expected version ">=16". Got "14.21.3" ```

The resolution on our side was to pin commander to a previous version(11.x > 10.x).

```
[1/4] 🤔  Why do we have the module "commander"...?
[2/4] 🚚  Initialising dependency graph...
warning No license field
warning Resolution field "commander@10.0.1" is incompatible with requested version "commander@^11.0.0"
warning Resolution field "canvas@2.6.1" is incompatible with requested version "canvas@^1.6.13"
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "commander@10.0.1"
info Reasons this module exists
   - "@vue#test-utils#pretty#js-beautify#editorconfig" depends on it
   - Hoisted from "@vue#test-utils#pretty#js-beautify#editorconfig#commander"
info Disk size without dependencies: "212KB"
info Disk size with unique dependencies: "212KB"
info Disk size with transitive dependencies: "212KB"
info Number of shared dependencies: 0
✨  Done in 0.12s.```

**Comment by cexbrayat** [comment]
Hi @Sham-Hulla 

I don't think the commit you link is the culprit: it landed this morning and is not yet released. And prettier is a dev dependency of VTU, so it should not affect you.
The error you get mentions commander, which is not a dependency of VTU, but a dependency of a dependency of a dependency of VTU, so there is nothing we can do to help.
Pinning the version is probably the right move if you need to support deprecated node versions.
